### PR TITLE
Refactor nailed get config

### PR DESCRIPTION
**Pull request by kkaempf**

### CODE REVIEW

**Comment by kkaempf** [comment]
Fixed TOPLEVEL commit

File: lib/nailed/github.rb [hunk 1]
```diff
@@ -1,14 +1,75 @@
...
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```
Comment (MaximilianMeister) [inline lib/nailed/github.rb]: as I said in #29 i would keep `fill_db_after_migration` in `lib/nailed.rb` as it doesn't have a GitHub only context.

one could split it later on to `bugzilla` and `github` methods.

**Comment by MaximilianMeister** [comment]
and can you please fix the old references to `get_config`?

```
lib/nailed/bugzilla.rb:4:      Bicho.client = Bicho::Client.new(Nailed.get_config["bugzilla"]["url"])
lib/nailed/jenkins.rb:4:      if Nailed.get_config["debug"]
lib/nailed/jenkins.rb:12:         :server_ip => Nailed.get_config["jenkins"]["server_ip"],
lib/nailed/jenkins.rb:13:         :username  => Nailed.get_config["jenkins"]["username"],
lib/nailed/jenkins.rb:14:         :password  => Nailed.get_config["jenkins"]["api_token"],
```

File: lib/nailed/github.rb (hunk 1 above)
Comment (kkaempf) [inline lib/nailed/github.rb]: Well, right now it gets a `github_context` passed, which is confusing. But I see your point, will revert it for now, and split it asap.

**Comment by kkaempf** [comment]
Fixed and pushed

**Comment by MaximilianMeister** [comment]
:+1:

**Pull request merged**